Sources/LanguageServerProtocol/SupportTypes/Playground.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// A playground item that can be used to identify playgrounds alongside a source file.
```

Comment:
Can you make this match the doc comments in `LSP Extensions.md`? Same for the properties.

Also could you indicate this as an LSP extension like we do for other types?